src/ICeWall/test/tcl_interface.example.tcl

```diff
@@ -0,0 +1,442 @@
+add_pad -edge left   -inst_name u_v18_30  -signal DVDD_0               -type vdd   -location {centre {x  105.000 y 1057.000}} -bondpad {centre {x 149.893 y 1057.000}}
+add_pad -edge left   -inst_name u_vss_30  -signal VSS                  -type vssio -location {centre {x  105.000 y  983.000}} -bondpad {centre {x 149.893 y  983.000}}
+add_pad -edge left   -inst_name u_vdd_30  -signal VDD                  -type vddio -location {centre {x  105.000 y  909.000}} -bondpad {centre {x 149.893 y  909.000}}
+add_pad -edge left   -inst_name u_vzz_31  -signal DVSS_0               -type vss   -location {centre {x  105.000 y  837.000}} -bondpad {centre {x 149.893 y  837.000}}
+add_pad -edge left   -inst_name u_v18_31  -signal DVDD_0               -type vdd   -location {centre {x  105.000 y  763.000}} -bondpad {centre {x 149.893 y  763.000}}
+add_pad -edge left   -inst_name u_vss_31  -signal VSS                  -type vssio -location {centre {x  105.000 y  689.000}} -bondpad {centre {x 149.893 y  689.000}}
+add_pad -edge left   -inst_name u_vdd_31  -signal VDD                  -type vddio -location {centre {x  105.000 y  615.000}} -bondpad {centre {x 149.893 y  615.000}}
+add_pad -edge left   -inst_name u_vzz_32  -signal DVSS_0               -type vss   -location {centre {x  105.000 y  504.000}} -bondpad {centre {x 149.893 y  504.000}}
+add_pad -edge left   -inst_name u_v18_32  -signal DVDD_0               -type vdd   -location {centre {x  105.000 y  430.000}} -bondpad {centre {x 149.893 y  430.000}}
+add_pad -edge left   -inst_name u_vss_32  -signal VSS                  -type vssio -location {centre {x  105.000 y  356.000}} -bondpad {centre {x 149.893 y  356.000}}
+add_pad -edge left   -inst_name u_vdd_32  -signal VDD                  -type vddio -location {centre {x  105.000 y  282.000}} -bondpad {centre {x 149.893 y  282.000}}
+add_pad -edge left   -inst_name u_cbrk14               -type cbk   -location {centre {x  105.000 y 2772.500}}
+add_pad -edge left   -inst_name u_cbrk15               -type cbk   -location {centre {x  105.000 y 2145.500}}
+add_pad -edge left   -inst_name u_cbrk16               -type cbk   -location {centre {x  105.000 y 1518.500}}
+add_pad -edge left   -inst_name u_cbrk17               -type cbk   -location {centre {x  105.000 y  891.500}}
+add_pad -edge left   -inst_name u_cbrk18               -type cbk   -location {centre {x  105.000 y  227.500}}
+add_pad -edge left   -inst_name u_pwrdet13             -type pdt   -location {centre {x  105.000 y 2459.000}}
+add_pad -edge left   -inst_name u_pwrdet14             -type pdt   -location {centre {x  105.000 y 1832.000}}
+add_pad -edge left   -inst_name u_pwrdet15             -type pdt   -location {centre {x  105.000 y 1205.000}}
+add_pad -edge left   -inst_name u_pwrdet16             -type pdt   -location {centre {x  105.000 y  541.000}}
+
+if {[catch {initialize_padring} msg]} {
+  puts $errorInfo
+  puts $msg
+  return
+}
```

Comment:
I would just make this `initialize_padring`

Reply:
Done